code/ship/shipfx.cpp

```diff
@@ -106,7 +106,7 @@ static void shipfx_subsystem_maybe_create_live_debris(object *ship_objp, ship *s
 
 	// copy angles
 	angles copy_angs = pm->submodel[submodel_num].angs;
-	angles zero_angs = {0.0f, 0.0f, 0.0f};
+	angles zero_angs = vmd_zero_angles;
```

Comment:
Delete this line and plug in vmd_zero_angles directly into line 145
`pm->submodel[submodel_num].angs = zero_angs;`

Reply:
Everything should be fixed now!